homeassistant/components/iron_os/coordinator.py

```diff
@@ -22,6 +28,7 @@ class IronOSCoordinator(DataUpdateCoordinator[LiveDataResponse]):
     """IronOS coordinator."""
 
     device_info: DeviceInfoResponse
+    settings: SettingsDataResponse = SettingsDataResponse()
```

Comment:
What's this?

Reply:
Not sure, I did this some months ago.
Maybe `settings: SettingsDataResponse = {}`  would be better 🤔